components/ws-manager-api/README.md

```diff
@@ -0,0 +1,6 @@
+## Overview
+The ws-manager-api component hosts the api component of ws-manager.
```

Comment:
are the terms protocol and api interchangeable? I have used them as interchangeable. LMK if other wise.